_benchmark/workloads/index.md

```diff
@@ -0,0 +1,249 @@
+      "clients": 8
+    },
+    {
+      "operation": {
+        "name": "query-match-all",
+        "operation-type": "search",
+        "body": {
+          "query": {
+            "match_all": {}
+          }
+        }
+      },
+      "iterations": 1000,
+      "target-throughput": 100
+    }
+  ]
+}
+```
+
+According to this schedule, each action will run in the following order.
+
+1. The `create-index` operation creates an index with benchmarked data.
+2. The `cluster-health` operation assess the health of the cluster before running the workload. In this example, the workload will wait until the status of cluster's health is `green`.
+3. The `bulk` operation run the `bulk` API to index `500` documents at once.
+4. Before benchmarking, the workload waits until the specified `warmup-time-period` passes. In this example, the warmup period if `120` seconds.
+5. The `clients` options defines the number of clients that will run the remaining actions in the schedule concurrently.
+6. The `search` runs a `match_all` query to find documents that match each other after they are indexed by the `bulk` API using the 8 clients specified.
+7. The `iterations` options indicates the number of time each client runs the `search` operation. The report generated by the benchmark automatically adjusts the percentile numbers based on this number. To generate a precise percentile, the benchmark needs to run at least 1000 iteration.
+8. Lastly, the `target-throughput` defines the number of requests per second each client performs, which when set can help reduce the latency of the benchmark. For example, a `target-throughput` of 100 requests divided by 8 clients means that each client will issue 12 requests per second.
+
+
+## Workload examples
+
+If you want to try certain workloads before creating your own, use the following examples.
+
+### Running unthrottled
+
+In the following example, OpenSearch Benchmark runs a bulk index operation unthrottled for one hour against the `movies` index:
+
+```json
+{
+  "description": "Tutorial benchmark for OpenSearch Benchmark",
+  "indices": [
+    {
+      "name": "movies",
+      "body": "index.json"
+    }
+  ],
+  "corpora": [
+    {
+      "name": "movies",
+      "documents": [
+        {
+          "source-file": "movies-documents.json",
+          "document-count": 11658903, # Fetch document count from command line
+          "uncompressed-bytes": 1544799789 # Fetch uncompressed bytes from command line
+        }
+      ]
+    }
+  ],
+  "schedule": [
+  {
+    "operation": "bulk",
+    "warmup-time-period": 120,
+    "time-period": 3600,
+    "clients": 8
+  }
+]
+}
+```
+
+### Workload with a single task
+
+The following workload runs a benchmark with a single task, a `match_all` query. According to the `schedule`, the workload runs the `match_all` query at 10 operations per second with one client, uses 100 iterations to warmup, and the next 100 iterations to measure the benchmark:
```

Comment:
Maybe clarify how we deduced that it has one client? 
"Since there's no `clients` parameter specified in the schedule, we can assume that there is only one client being used"